pyk/src/pyk/kllvm/hints/prooftrace.py

```diff
@@ -38,7 +44,8 @@ class LLVMRewriteEvent(LLVMStepEvent):
     @property
     @abstractmethod
     def rule_ordinal(self) -> int:
-        """Returns the axiom ordinal number of the rewrite rule. The rule ordinal represents the `nth` axiom in the kore definition."""
+        """Returns the axiom ordinal number of the rewrite rule. The rule ordinal represents the `nth` axiom in the kore
+        definition."""
```

Comment:
```suggestion
        """Return the axiom ordinal number of the rewrite rule.
        
        The rule ordinal represents the `nth` axiom in the kore definition.
        """
```

https://peps.python.org/pep-0257/